pkg/platform/aws/template.go

```diff
@@ -16,7 +16,7 @@ package aws
 
 var terraformConfigTmpl = `
 module "aws-{{.Config.ClusterName}}" {
-  source = "../lokomotive-kubernetes/aws/flatcar-linux/kubernetes"
+  source = "../terraform-modules/aws/flatcar-linux/kubernetes"
```

Comment:
I wonder if Terraform is fine with such changes during the upgrades.

Reply:
On my testing it works fine as long as you run `terraform init` before the re-apply, which we do.